docs/vision-usage.rst

```diff
@@ -169,9 +169,9 @@ will also provide their known geographical locations if available.
     -33.857123
     >>> landmarks[0].locations[0].longitude
     151.213921
-    >>> landmarks[0].bounding_poly.vertices[0].x_coordinate
+    >>> landmarks[0].bounds.vertices[0].x_coordinate
     78
-    >>> landmarks[0].bounding_poly.vertices[0].y_coordinate
+    >>> landmarks[0].bounds.vertices[0].y_coordinate
```

Comment:
I was confused here that these were instances of `face.Landmark`, which have no `bounds` attribute:  but these are `entity.EntityAnnotation` instances, right?  Is it just that the API reuses the term confusingly, and we follow suit?